core/dictgen/src/rootcling_impl.cxx

```diff
@@ -143,6 +143,17 @@ namespace genreflex {
 
 ////////////////////////////////////////////////////////////////////////////////
 
+static llvm::cl::OptionCategory gRootclingOptions("rootcling common options");
+
+ // FIXME: We should remove the IgnoreExistingDict option as it is not used.
+static llvm::cl::opt<bool>
+gOptIgnoreExistingDict("r",
+               llvm::cl::desc("Deprecated, legacy flag which is ignored."),
```

Comment:
This is very confusing ... if it is "ignored" why does it have a very visible effect (no dictionary generated)?

Reply:
In this PR, I did not write that comment, but I just moved that line along with the comment up from here - https://github.com/root-project/root/pull/5223/files#diff-64a4c9a2b092d1b125517e200783ca9fL3683. I think the comment is there because of some future plans to deprecate the -r flag? @vgvassilev

Reply:
Yes, we should update the comment. It is inaccurate but it is not introduced in this PR.